benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -602,26 +602,24 @@ msgstr ""
 "para cerrar la sesión de"
 
 msgid "enrollment.pages.index.link_card_item.heading"
-msgstr "El siguiente paso es vincular su tarjeta bancaria"
-
-#, python-format
-msgid "enrollment.pages.index.link_card_item.p[0]%(link)s"
 msgstr ""
-"Ahora que se ha verificado su elegibilidad, vincularemos su descuento de "
-"tránsito a su tarjeta a través de nuestro socio de pago, <a "
-"href=\"%(link)s\">Littlepay</a>. No almacenamos su información y no se te "
-"cobrará."
+"El siguiente paso es conectar su tarjeta sin contacto a su beneficio de "
+"tránsito"
 
-msgid "enrollment.pages.index.link_card_item.p[1]"
-msgstr ""
-"Por favor utilice una tarjeta de crédito o débito Visa o Mastercard sin "
-"contacto emitida por un banco."
+msgid "enrollment.pages.index.link_card_item.p[0]s[0]"
+msgstr "Será dirigido a nuestro socio de pago, "
+
+msgid "enrollment.pages.index.link_card_item.p[0]s[1]"
+msgstr "No almacenamos su información y no se le cobrará."
+
+msgid "enrollment.pages.index.link_card_item.p[1]s[0]"
+msgstr "Utilice una tarjeta de débito o crédito de Visa o Mastercard.."
```

Comment:
@thekaveman Directly added new Spanish copy into this PR!

Reply:
<img width="1512" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/37e8d4ff-db1c-4e58-a496-bbebbc32612f">